Tests Authenticated Task

alejandropiuryp opened the pull request:
Se ha creado la clase AuthenticatedTaskListAndShowTests.java donde se han realizado todos los test relacinados con la feature con un coverage total del 74,1%.

alejandropiuryp committed:
Tests Authenticated Task

Se ha creado la clase AuthenticatedTaskListAndShowTests.java donde se han realizado todos los test relacinados con la feature con un coverage total del 74,1%.

alvechdel reviewed (changes requested):
Check order of tests and also exception thrown on non-authorized tests.

alejandropiuryp committed:
Tests negativos arreglados

Se ha arreglado los test negativos de la feature reemplazando el método navigate por el drive, de esta forma da el error de autorización correspondiente

alejandropiuryp committed:
Merge branch 'develop' into Test-Authenticated-Task

manpercar1 committed:
Merge branch 'develop' into Test-Authenticated-Task

alvechdel reviewed (approved):
Now is all working correctly, good job.
